manual/tracy.tex

```diff
@@ -3542,9 +3542,45 @@ \subsubsection{Timeline view}
 \item \emph{Gray} -- Threads assigned to other programs running in the system.
 \end{itemize}
 
-When the \faMousePointer{}~mouse pointer is hovered over either the CPU data zone or the thread timeline label, Tracy will display a line connecting all zones associated with the selected thread. This can be used to quickly see how the thread migrated across the CPU cores.
+When the \faMousePointer{}~mouse pointer is hovered over either the CPU data zone or the thread timeline label, Tracy will display a line connecting all zones associated with the selected thread. This can be used to quickly see how the thread migrated across the CPU cores. 
 
-Clicking the \LMB{}~left mouse button on a tracked thread will make it visible on the timeline if it was either hidden or collapsed before.
+It will also add lines starting with a filed circle to denote wake up events. Those are useful to pinpoint the origin of a thread waking up, especially when holding locks. It may also start from an empty region, denoting the time at which the kernel made chose to schedule or boost the priority of your thread. Wake ups will have a different color based on the reason for which the thread was waiting to be scheduled. 
```

Comment:
> Wake ups will have a different color based on the reason for which the thread was waiting to be scheduled. 

Is there a way to know what these colors will be?

Reply:
Right now not really, but the "wait reason" can be seen from the thread lanes. The idea behind coloring it was to quickly see the different kinds of wait reason, but I think in the long term we might want to categorize each wait reason and regroup them ?